### DISCUSSION TITLE
"see also" links on JanetDocs?

### DISCUSSION THREAD

**Discussion by uvtc** [janet-lang/janet#744] [General]
It looks like a "see also" feature was at one time added to [JanetDocs](https://janetdocs.com/) via https://github.com/swlkr/janetdocs/pull/27 , but I don't see a "see also" link anywhere there. Was it added and then removed? @swlkr ? @MikeBeller ?

**Comment by sogaiu** [comment]
@uvtc I found a couple of places that have "see also" stuff:

* https://janetdocs.com/sorted
* https://janetdocs.com/find

I don't know how those get added.

**Reply by uvtc** [reply]
Interesting! Thanks, @sogaiu . Created https://github.com/swlkr/janetdocs/issues/40 to keep track of this.

**Comment by pepe** [comment]
It is the part of the code in seed: https://github.com/swlkr/janetdocs/blob/master/seed.janet, so any new links need to be added there.
